gui/src/main/style.css

```
@@ -0,0 +1,110 @@
+    --color-highlight: #95F3F6;
+    --color-divider: rgba(149, 243, 246, 0.075);
+    --enth-only: 1;
+    --app-font: "League Spartan", sans serif;
+  }
+
+  body {
+    @apply overflow-hidden main-background font-app {}
+  }
+
+  #root {
+    @apply flex {}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)